app/uk/gov/hmrc/gform/gform/CompositeAuthController.scala

```diff
@@ -50,11 +51,21 @@ class CompositeAuthController(
     )
   )
 
-  def authSelectionForm(formTemplateId: FormTemplateId, ggId: Option[String], continue: String) =
+  def authSelectionForm(
+    formTemplateId: FormTemplateId,
+    ggId: Option[String],
+    continue: String,
+    se: SuppressErrors
+  ) =
     nonAutheticatedRequestActions.async { implicit request => implicit lang =>
       val formTemplate = request.attrs(FormTemplateKey)
       val compositeAuthFormPage =
-        new CompositeAuthFormPage(formTemplate, choice, ggId)
+        choice
+          .bindFromRequest()
```

Comment:
Is this binding needed? Will it ever bind successfully given this is a GET endpoint?